packages/airnode-validator/src/config/config.ts

```diff
@@ -5,9 +5,12 @@ import { version as packageVersion } from '../../package.json';
 import { OIS, oisSchema } from '../ois';
 import { SchemaType } from '../types';
 
+export const evmAddressSchema = z.string().regex(/^0x[a-fA-F0-9]{40}$/);
+export const evmIdSchema = z.string().regex(/^0x[a-fA-F0-9]{64}$/);
+
 export const triggerSchema = z
   .object({
-    endpointId: z.string(),
+    endpointId: evmIdSchema,
```

Comment:
Does this actually make sense? We're not enforcing the endpoint ID to be constructed from the endpoint name and OIS title (even though we encourage it in the docs), why force the format at all? I think it can be a random string and it will work, right? At least that's what docs are saying.

`This convention of determining an endpointId is not enforced at the protocol-level. For example, one could choose to generate an endpointId randomly, and as long as requesters use the correct endpointId, this will not be an issue.`

Reply:
It was part of the legacy validator rules, but yeah we might drop this. I personally do not care about this much and it can be a validator warning later.

Reply:
Yeah it would be nice to have this as a warning